app/services/processors/blog_post_count_metrics_remover.rb

```diff
@@ -0,0 +1,9 @@
+module Processors
+  class BlogPostCountMetricsRemover < Processors::Base
+    private
+
+    def process
+      Metric.where(name: Metric.names[:blog_post_count]).destroy_all
+    end
+  end
+end
```

Comment:
Is this something that are we going to use more than once? Otherwise, why don't you put the code inside the rake?

Reply:
I hadn't thought about that difference, I just always assumed it was better to just write the code outside of the task.
Indeed it's gonna be run just once. I planned to remove this code and the task as well anyway when I finished using it, though. Should I still move the code?